src/Terrajobst.ApiCatalog/MissingNetFxFeatures/MissingNetFxFeatures.xml

```diff
@@ -587,6 +587,30 @@
         </appliesTo>
     </feature>
 
+    <feature name="System.Runtime.InteropServices.GetActiveObject()">
+        <description>
+          The Marshal.GetActiveObject() API is a wrapper over the Running Object Table (ROT, see
+          https://docs.microsoft.com/windows/win32/com/registering-objects-in-the-rot) and relies
+          on specific Win32 APIs that can be called via a P/Invoke. An example of how to manually
+          communicate with the ROT can be found at
+          https://github.com/AaronRobinsonMSFT/COMInterop/blob/master/OutOfProcDemo.
+        </description>
+        <appliesTo>
+            <member namespace="System.Runtime.InteropServices" type="Marshal" name="GetActiveObject()" />
+        </appliesTo>
+    </feature>
+
+    <feature name="System.Runtime.InteropServices Type Library Support">
+        <description>
+            Dynamic generation and interaction with type libraries isn't available on .NET Core.
+        </description>
+        <appliesTo>
+            <member namespace="System.Runtime.InteropServices" type="Marshal" name="GetTypeForITypeInfo(IntPtr)" />
+            <member namespace="System.Runtime.InteropServices" type="Marshal" name="GetTypeLibName(ITypeLib)" />
+            <member namespace="System.Runtime.InteropServices" type="Marshal" name="GetTypeLibName(UCOMITypeLib)" />
```

Comment:
@AaronRobinsonMSFT Are there other APIs that should be included here?

Reply:
Those are the ones that come to mind.